src/components/Projects/MyProjects.tsx

```diff
@@ -0,0 +1,68 @@
+import { t, Trans } from '@lingui/macro'
+import Grid from 'components/shared/Grid'
+import Loading from 'components/shared/Loading'
+import ProjectCard from 'components/shared/ProjectCard'
+import { NetworkContext } from 'contexts/networkContext'
+import { ThemeContext } from 'contexts/themeContext'
+import { useMyProjectsQuery } from 'hooks/v1/Projects'
+import React, { useContext } from 'react'
+import { InfoCircleOutlined } from '@ant-design/icons'
+
+export default function MyProjects() {
+  const { userAddress } = useContext(NetworkContext)
+
+  const { data: projects, isLoading } = useMyProjectsQuery(userAddress)
+
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  return (
+    <React.Fragment>
+      {projects && projects.length > 0 && (
+        <React.Fragment>
+          <Grid>
+            {projects.map(p => (
+              <ProjectCard project={p} />
+            ))}
+          </Grid>
+        </React.Fragment>
+      )}
+
+      {!isLoading &&
+        projects &&
+        (projects.length === 0 ? (
+          <div
+            style={{
+              textAlign: 'center',
+              color: colors.text.disabled,
+              padding: 20,
+            }}
+            hidden={isLoading}
+          >
+            <Trans>You haven't created any projects yet.</Trans>
```

Comment:
**praise** Love!

What you think about adding a button here to create a project? Literally

`Create a project` => links to create page

Reply:
The one tricky thing is that v1 doesn't have a 'create project' page and this is a shared component. It's easy enough for v2 but for v1 we'd be linking to the main page then scrolling to the create section?

Reply:
Few things:
- we'll still have a `/v1/create` route for v1. V2 will be at `/create`
- No Create UI on the homepage anymore
- Everything is v2-by-default going forward.

I think we can leave this for now though - save ourselves some hassle of dealing with a feature flag here, and add a button once V2 is out 🙏